src/steamship/agents/schema/action.py

```diff
@@ -22,6 +22,12 @@ class Action(BaseModel):
     output: Optional[List[Block]]
     """Any direct output produced by the Tool."""
 
+    return_direct: bool = False
```

Comment:
curious: should we label this more like  `is_final` or `tool_is_direct` similar to track that state (and update the comment -- as i'm not expecting users to set `return_direct` on `Action`s) ?

aside: Would this be "simpler" if the `Action` returned by an `Agent`/`AgentExecutor` when  `Tool.return_direct == True` was a `FinishAction` instead?  Or even a new type of `Action` (ex: `DirectReturnAction`) ?  Maybe that would be more complex (as the `AgentExecutor` would have to be involved) ??